opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanBuilderShim.java

```diff
@@ -121,27 +121,24 @@ public SpanBuilder withTag(String key, String value) {
           break;
       }
     } else if (Tags.ERROR.getKey().equals(key)) {
-      if (Boolean.parseBoolean(value)) {
-        error = true;
-      }
-    } else {
-      this.spanBuilderAttributeKeys.add(key);
-      this.spanBuilderAttributeValues.add(AttributeValue.stringAttributeValue(value));
+      error = Boolean.parseBoolean(value);
     }
 
+    this.spanBuilderAttributeKeys.add(key);
+    this.spanBuilderAttributeValues.add(AttributeValue.stringAttributeValue(value));
```

Comment:
why do you need this? I was expecting that when the backend is implementing the new API they can record this for backwards compatibility, don't need to do it here.

OT (span.kind tag) -> OpenConsensus (Span.Kind property) -> Backend(span.kind tag/attribute)